go.mod

```
@@ -0,0 +1,3 @@
+module github.com/MrNemo64/ClashKingProxy
```

```suggestion
module github.com/ClashKingInc/ClashKingProxy
```
Probably want to change this